pkg/internal/controller/metrics/metrics.go

```diff
@@ -48,4 +48,10 @@ func init() {
 	metrics.Registry.MustRegister(QueueLength)
 	metrics.Registry.MustRegister(ReconcileErrors)
 	metrics.Registry.MustRegister(ReconcileTime)
+	metrics.Registry.MustRegister(
+		// expose process metrics like CPU, Memory, file descriptor usage etc.
+		prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}),
+		// expose Go runtime metrics like GC stats, memory stats etc.
+		metrics.Registry.MustRegister(prometheus.NewGoCollector()),
```

Comment:
Looks like you don't need to wrap `prometheus.NewGoCollector()` in this `MustRegister`
```suggestion
		prometheus.NewGoCollector(),
```

Reply:
Good catch, last minute refactor introduced it. I had two separate MustRegister calls earlier.